src/utils/proposalFields.ts

```diff
@@ -1,19 +1,17 @@
-import type { BaseField } from '@pdc/sdk';
-import { ApiProposal } from '../pdc-api';
+import type { Proposal, BaseField } from '@pdc/sdk';
 import { PROPOSAL_NAME_CASCADE, PROPOSAL_NAME_FALLBACK } from './proposals';
 
-const mapProposalBaseFields = (
-	baseFields: BaseField[],
-	proposal: ApiProposal,
-) =>
+const mapProposalBaseFields = (baseFields: BaseField[], proposal: Proposal) =>
 	(proposal.versions[0]?.fieldValues ?? []).map(
 		({ applicationFormField, value }) => {
 			const baseField = baseFields.find(
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
more explicit ignore lines

Reply:
applicationFormField is potentially of type 'any'

Reply:
When you come across things like this please consider opening an issue in the sdk repository (our SDK shouldn't be resulting in eslint warnings!)

I'm not sure why `any` is listed as a possible value but that's definitely a bug!

Reply:
Heard loud and clear, opening an issue now! To that end then I will also remove the draft status to this PR(as I believe this was the only reason I had left as draft)

Reply:
Reminder to add a comment explaining the reason for the eslint disable / linking to the opened issue